### DISCUSSION TITLE
KusionStack Open Source Quick News in September

### DISCUSSION THREAD

**Discussion by He1pa** [KusionStack/community#34] [Show and tell]
+ [KusionStack 英文版](https://kusionstack.io/) 上线，欢迎大家反馈
+ [Konfig](https://github.com/KusionStack/konfig) 模型库 https://github.com/KusionStack/konfig/pull/38 （感谢 @JYSDeveloper 提供的诸多反馈）
+ Kusion Engine架构升级，统一管理 TF 与 K8s 两种资源类型，进度 90%
+ KCLVM 发布 [v0.4.3 版本](https://github.com/KusionStack/KCLVM/releases/tag/v0.4.3-alpha.1)
+ Kusion 加入 [CNCF Landscape](https://l.cncf.io/) (Automation & Configuration)，加入开源社区只是第一步，继续加油
+ CCF GLCC 夏令营完成中期考核，并和同学对中后期的工作进行交流沟通
+ KCL论文被 SETTA 2022 接受

KusionStack 是开源的云原生可编程技术栈，灵感源于融合（Fusion）一词，旨在帮助企业构建的应用运维配置管理平面及 DevOps 生态。基于 Platform as Code （平台服务即代码）理念，研发者可以快速收敛围绕应用运维生命周期的全量配置定义，面向混合技术体系及云环境，完成从应用运维研发到上线的端到端工作流程，真正做到一处编写，随处交付。

**Comment by Peefy** [comment]
KusionStack Open Source Quick News in September

+ Welcome to our new [KusionStack Site](https://kusionstack.io/)
+ Adding ClickHouse [Konfig](https://github.com/KusionStack/konfig) Models https://github.com/KusionStack/konfig/pull/38 （Thanks for the feedback of @JYSDeveloper）
+ Kusion Engine architecture upgrade, unified management of TF and K8s resource types, 90% progress
+ KCLVM [v0.4.3 Release](https://github.com/KusionStack/KCLVM/releases/tag/v0.4.3-alpha.1)
+ KusionStack joins [CNCF Landscape](https://l.cncf.io/) (Automation & Configuration)
+ Complete the mid-term assessment of CCF GLCC summer camp, and communicate with students on the work in the middle and later stages
+ KCL paper accepted by SETTA 2022

KusionStack is a highly flexible programmable technology stack to enable unified application delivery and operation, inspired by the word Fusion, which aims to help enterprises build an application-centric configuration management plane and DevOps ecosystem. Based on the concept of Platform as Code, developers can quickly unify the full configuration schemas, constraints, policies, and configs across the application life cycle, work with the hybrid technologies and cloud environment, go through the end-to-end workflow from programming to launching, and truly achieve write once, deliver anywhere.
